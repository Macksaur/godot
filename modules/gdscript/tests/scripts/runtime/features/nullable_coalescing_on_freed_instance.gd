func test():
    var node := Node.new()
    node.free()
    node = node ?? Node.new()
    print(node?.get_parent())
    node.free()