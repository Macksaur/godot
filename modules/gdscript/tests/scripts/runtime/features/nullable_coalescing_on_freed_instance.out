GDTEST_OK
<Object#null>
